Makefile

```diff
@@ -160,7 +160,7 @@ check-all-generated-files-up-to-date: generate-all ## Run the generate all comma
 ### Update maxpods.go from AWS
 .PHONY: update-maxpods
 update-maxpods: ## Re-download the max pods info from AWS and regenerate the maxpods.go file
-	@cd pkg/nodebootstrap && go run maxpods_generate.go
+	@cd pkg/nodebootstrap/legacy && go run maxpods_generate.go
```

Comment:
This will fail after https://github.com/weaveworks/eksctl/pull/4968/files.

I removed that entire folder. Any reasons why this is needed?

Reply:
Err... Not the entire folder, but significant parts of it.

Reply:
Yep, `maxpods_generate.go` has been removed from this folder. So this will break. :)

Reply:
Oh if it's planning to be removed now I can just delete that job from the workflow

Reply:
That sounds good!